requirements.txt

```diff
@@ -50,4 +49,4 @@ Unidecode==1.1.1
 urllib3>=1.26.5
 websocket-client==0.56.0
 websocket-server==0.4
-jsonlines==1.2.0
+jsonlines==1.2.0
```

Comment:
can we put this back in alphabetical order?

Reply:
Pseudo-alphabetical order lol. there are some intentional non-monotonic places

Reply:
ok yeah i think we might've broken some stuff with the order wrong... sorry @mojtaba-komeili could you just revert to what it was before?